src/components/sticky/sticky.js

```diff
@@ -169,22 +171,22 @@ function MdSticky($document, $mdConstant, $compile, $$rAF, $mdUtil) {
         setCurrentItem(null);
 
       // Going to next item?
-      } else if (isScrollingDown && self.next) {
-        if (self.next.top - scrollTop <= 0) {
+      } else if (isScrollingDown) {
```

Comment:
It feels like this whole block could be a better high-level comment explaining what's going on.